### ISSUE TITLE
Deploy to Netlify

### ISSUE THREAD

**Issue by tomasgil123** [thetre97/gridsome-plugin-flexsearch#2]
Im using gridsome-plugin-flexsearch and it works locally, but when I try to deploy it to Netlify I get the following error:
![image](https://user-images.githubusercontent.com/14185059/69808235-d9e34080-11c5-11ea-8f6c-438633c128c1.png)

Im including gridsome-plugin-flexsearch like this in gridsome.config.js

```
plugins: [
      {use: 'gridsome-plugin-flexsearch',
    options: {
    collections: [
        {
            typeName: 'BookEntry',
            indexName: 'BookEntry',
            fields: ['id', 'title', 'access']
        },

    ],
        searchFields: ['title']
}}
,
```

Any ideas to solve it?

**Comment by thetre97** [comment]
Could I check what version of gridsome + this plugin you are using?

**Comment by tomasgil123** [comment]
These are my dependencies:
```
"dependencies": {
    "@gridsome/plugin-google-analytics": "^0.1.0",
    "@gridsome/plugin-sitemap": "^0.2.1",
    "@gridsome/remark-prismjs": "0.0.6",
    "@gridsome/source-filesystem": "^0.6.0",
    "@gridsome/transformer-remark": "^0.3.4",
    "axios": "^0.19.0",
    "gridsome": "^0.7.0",
    "gridsome-plugin-algolia": "^2.1.1",
    "gridsome-plugin-flexsearch": "^0.1.5",
    "netlify-identity-widget": "^1.5.5",
    "typeface-source-sans-pro": "^0.0.54",
    "vue-feather-icons": "^4.22.0",
    "vuex": "^3.1.1"
  },
  "devDependencies": {
    "node-sass": "^4.12.0",
    "sass-loader": "^7.1.0",
    "style-resources-loader": "^1.2.1"
  }
```

**Comment by thetre97** [comment]
Thanks - I think it is probably because gridsome changed the api recently (from `config.outDir` to `config.outputDir`) so I will add a fallback for that.

**Comment by tomasgil123** [comment]
ok, thanks!

**Comment by thetre97** [comment]
Just published v0.1.6... Let me know if that fixes it?

**Comment by tomasgil123** [comment]
yep, it worked. Thanks man!
